src/App.jsx

```diff
@@ -14,45 +16,89 @@ export const goodsFromServer = [
   'Garlic',
 ];
 
-export const App = () => (
-  <div className="section content">
-    <div className="buttons">
-      <button
-        type="button"
-        className="button is-info is-light"
-      >
-        Sort alphabetically
-      </button>
-
-      <button
-        type="button"
-        className="button is-success is-light"
-      >
-        Sort by length
-      </button>
-
-      <button
-        type="button"
-        className="button is-warning is-light"
-      >
-        Reverse
-      </button>
-
-      <button
-        type="button"
-        className="button is-danger is-light"
-      >
-        Reset
-      </button>
-    </div>
+const SORT_TYPE_AB = 'name';
+const SORT_TYPE_LENGTH = 'length';
+
+function getPreparedGoods(goods, { sortType, reverse } = {}) {
+  let preparedGoods = [...goods];
+
+  if (sortType) {
+    preparedGoods = preparedGoods.sort((firstGood, secondGood) => {
```

Comment:
don't need to reset the preparedGoods list, because the method sort() already do it